src/components/lib/pubsub.js

```diff
@@ -186,10 +186,3 @@ export function getInstance() {
 }
 
 export var PubSub = getInstance();
-
```

Comment:
I'm almost certain this is critically important.

Reply:
maybe I should have left a comment to that effect. I think this handles either the old-style node imports or raw ES6... I don't quite remember. If this is deleted, possibly we'd want to delete it in a branch with Vite builds.

Reply:
Added back in.